<launch>

<arg name="imu_topic" default="/imu_data"/>
<arg name="frame_id" default="imu"/>
=<arg name="fixed_frame" default="map"/>
<arg name="publish_tf" default="true"/>


  <node name="mpu6050" pkg="mpu6050_i2c" type="mpu6050_i2c.py" output="screen">
        <param name="imu_topic" type="string" value="/imu/data_raw"/>
        <param name="frame_id" type="string" value="$(arg frame_id)"/>
  </node>
  <node name="imu_filter_node" pkg="imu_filter_madgwick" type="imu_filter_node" >
    <param name="use_mag"  value="false"/>
    <param name="fixed_frame"  value="$(arg fixed_frame)"/>
    <param name="constant_dt"  value="0.0"/>

    <param name="publish_tf"  value="$(arg publish_tf)"/>
    <remap from="imu/data" to="$(arg imu_topic)"/>

  </node>
  <node pkg="tf" type="static_transform_publisher" name="imu_to_imu_link" args="0 0 0 0 0 0 imu imu_link 100"/>
</launch>